components/brave_rewards/browser/rewards_protocol_handler.cc

```diff
@@ -60,9 +60,12 @@ void LoadRewardsURL(
 
   // Only accept rewards scheme from allowed domains
   const std::string bitflyer_staging_host = GURL(BITFLYER_STAGING_URL).host();
+  const std::string gemini_staging_host = GURL(GEMINI_OAUTH_STAGING_URL).host();
```

Comment:
We should `DCHECK` `GURL` is valid using `is_valid`

Reply:
fixed: https://github.com/brave/brave-core/pull/8566/files#diff-a4347b499cd677506cc905c42cc1ce9c6b1155f595181c9de70b2a190843dccf